README.md

```diff
@@ -0,0 +1,122 @@
+# -mollify
+
+![Mollify Preview](readme/mollify.jpg)
+
+Mollify is a dynamic Learning Management System (LMS) designed to provide a user-friendly interface for managing and navigating markdown content. The platform offers a multitude of features, including content navigation, search capabilities, bookmarking, text-to-speech functionality, an AI assistant, and a markdown editor with 'What You See Is What You Get' (WYSIWYG) capabilities.
+
+## Description
+
+Mollify, divided into various packages within a Lerna monorepository, primarily consists of the LMS and CLI. These modules empower users to manage their content efficiently and transform it into a navigable webpage.
+
+### Mollify LMS Package
+
+The Mollify LMS package is the frontend of the Mollify project, constructed using Sveltekit, TypeScript, Tailwind, Skeleton UI, and MDsveX. The aim is to convert markdown content into an accessible website, enabling users to navigate, search, and bookmark content effectively, along with additional feature integration. The ultimate goal is to allow users to install the LMS package, migrate their existing content or create new, and finally, present it as a website.
+
+### Mollify CLI Package
+
+Mollify CLI is a command-line tool designed to manage the filesystem for the Mollify Learning Management System (LMS). It ensures that the file structure adheres to SvelteKit's file path conventions and provides a convenient way to create, archive, and remove various entities within the LMS. Mollify CLI is written in TypeScript and uses the Commander and Enquirer libraries to provide a user-friendly and intuitive interface.
+
+### Mollify Molly Package
+
+Mollify Molly is an OpenAI-powered teaching assistant, designed to answer student queries by accessing the lesson content.
```

Comment:
As I understood it, Molly can also assist the teachers, in which case it's worth adding it

Reply:
I've expanded it a little to include are future aim and other possible uses.